### DISCUSSION TITLE
How to implement Hot Reloading to the lib when playing in the playground?

### DISCUSSION THREAD

**Discussion by piscespieces** [fkhadra/react-toastify#735] [Q&A]
I just cloned the project hoping to give it a read, understand it, and if possible be able to contribute.

I'm currently running the playground in the `example` directory, and from there I'm able to make changes and see them taking place instantly (hot reloading), but when making changes to the lib (react-toastify) I don't get hot reloading. I have to kill the server and run it again if I wanted to see changes made locally to the lib in the playground.

Would appreciate any insights or ideas to get around it. Thanks!

**Comment by fkhadra** [comment]
Hey @piscespieces, I migrated to the latest version of parcel and I didn't have time yet to fix the hot reloading when modifying the library itself. I'll try to fix the issue this weekend if possible
